components/RadioButton/RadioButton.jsx

```diff
@@ -0,0 +1,13 @@
+const RadioButton = (props) => {
+    const { changed, id, isSelected, label, value } = props
+    return (
+        <div className='RadioButton'>
+            <input id={id} onChange={changed} value={value} type='radio' checked={isSelected} />
+            <label style={{ margin: '0 5px' }} htmlFor={id}>
```

Comment:
Please use CSS instead of inline styles

Reply:
Done

Reply:
RadioButton.module.css created inside styles folder